README.md

```diff
@@ -102,12 +102,18 @@ Next, mount the root filesystem and the EFI partition. (By default the EFI parti
 # mount /dev/disk/by-label/SYSTEM_DRV /mnt/boot
 ```
 
-Finally, run `nixos-install`. Note that this repository is symlinked at `/x1e-nixos-config` for convenience:
+Finally, run `nixos-install`. Note that this repository is available as `x1e-nixos-config` in the flake registry:
 
 ```console
-# nixos-install --root /mnt --no-channel-copy --no-root-password --flake $(readlink /x1e-nixos-config)#system
+# nixos-install --root /mnt --no-channel-copy --no-root-password --flake x1e-nixos-config#system
 ```
 
+Explanation:
+
+- `--no-channel-copy` disables copying channels. If you would like to use a non-flake config with channels (which I don't recommend), you should omit this option.
+- `--no-root-password` disables setting a root password. Omit this if you don't have any other way to get into the installed system and need a root password.
+- `--flake x1e-nixos-config#system` specifies the example flake based configuration to be used that comes with this repository. Note that if you have already [created your own NixOS configuration](#making-your-own-nixos-configuration) for the to-be-installed system, you can use that here. (Either by specifying `--flake /path/to/your/config#your-hostname` or by placing your non-flake config at `/mnt/etc/nixos/configuration.nix` and omitting this option. Importantly, `nixos-generate-config` is not expected to work.) You can always switch to your own configuration later if you use the example config.
```

Comment:
> You can always switch to your own configuration later if you use the example config.

This is not strictly true, as the example config disables channels. We should warn users here that if they want a non-flake config, they have to use it from the start.